.buildkite/pipeline.yml

```diff
@@ -40,8 +42,10 @@ steps:
   - label: Lint
     key: lint
     plugins:
+      - *git-partial-clone-plugin
       - *nvm_plugin
-      - *git-cache-plugin
+    env:
+      BUILDKITE_PLUGINS_ALWAYS_CLONE_FRESH: true
```

Comment:
That was another leftover for iterating on the plugin. The env var makes Buildkite checkout the plugin from scratch instead of using a Buildkite-managed cached version.

Done. Thanks for bringing this up.